partials/banner.hbs

```diff
@@ -1,5 +1,86 @@
 <a class="donation-banner" id="donation-banner-link" href={{t "donation-banner-url"}}>
     <p>
-        {{{t "Learn to code — <span>free 3,000-hour curriculum</span>"}}}
+        {{{t "Help our nonprofit pay for servers. <span>Donate Now</span>."}}}
     </p>
 </a>
+<script>
+    // Conditionally load stripe
+    const loadStripeList = ['en'];
+    console.log(loadStripeList.includes(`{{@site.lang}}`));
+    console.log(`{{@site.lang}}`);
+
+    if (loadStripeList.includes(`{{@site.lang}}`)) {
+
+        // Set keys based on environement
+        const stripePublicKey = location.hostname === "localhost" || location.hostname === "127.0.0.1" ?
```

Comment:
Don't we want a different key for staging?

Reply:
That's a good point. Seems like Stripe on freecodecamp.dev/donate is using a test key.

If the Stripe test key is the same for localhost and .dev, maybe this line could be something like:

```
const stripePublicKey = location.hostname === "localhost" ||
            location.hostname === "127.0.0.1" || 
            location.hostname === "www.freecodecamp.dev" ? 
            "pk_test_xM2wpbgtyotYqFCK6Ss6noxP008nxRyaE9" : "pk_live_E6Z6xPM8pEsJziHW905zpAvF";
```

Reply:
sure, I will take care of that.

Reply:
on a second thought, maybe I should just check for .org and return the live keys.

Reply:
Awesome, that sound much cleaner.